lib/content/write.js

```diff
@@ -67,6 +67,7 @@ class CacacheWriteStream extends Flush {
         this.cache,
         this.opts
       )
+      this.handleContentP.catch(error => this.destroy(error))
```

Comment:
`this.destroy` seems to be overreaching in scope.  I think we want `cb(error)`

We can also attach it directly to the function call `handleContent().catch()`

Reply:
Unfortunately this does not help.